CMakeLists.txt

```diff
@@ -19,6 +19,26 @@ message(STATUS "Configuration: ${CONFIG_OPTIONS}")
 
 project(openscad)
 
+# Default to Release build
+if(NOT CMAKE_BUILD_TYPE)
```

Comment:
Usually, one shouldn't edit the CMAKE_BUILD_TYPE, as cmake may be configured to e.g. build both release and debug configurations at the same time. I have to look into how to best achieve an optimized default build.

Reply:
As far as I understand it's only possible for build type to hold a single value.  You might be thinking about the 'RelWithDebInfo' (Release with debug info), build type regarding "both release and debug configurations at the same time"?  RelWithDebInfo is a default CMAKE_BUILD_TYPE that just specifies flags of: "-O2 -g -NDEBUG"

Also this code will only be set CMAKE_BUILD_TYPE if the user has not specified one.

Reply:
Try `cmake -G Xcode` or `cmake -g "Visual Studio 14 2015"`. That will generate both release and debug builds.